documentation/SupportingDocuments/Population Density API - Algorithm Initial Proposal.md

```diff
@@ -0,0 +1,33 @@
+# Population Density Data API
+## Algorithm initial proposal
+### Inputs for the algorithm:
+* UEs connection records data is received pseudonymized or hashed.
+* Requested Space is received in the API customers’ request.
+### Process is as follows:
+<ol>
+<li><strong>Cleanup</strong>: Repeated records will be deleted and traffic events associated with M2M lines are filtered due to they do not contribute to the process of estimating future population density.<br>
```

Comment:
as discussed since this is a high level easy to follow illustration let's avoid terms which have broad usage.
I.e. replace M2M lines with "IoT and non human wearing devices".

"Traffic events" is also a unclear term. What traffic is mentioned?